api/tests/test_room_store.py

```diff
@@ -244,3 +244,13 @@ async def test_write_if_missing_does_not_overwrite(room_store: RoomStore) -> Non
     await room_store.add_request(TEST_ROOM_ID, VALID_REQUEST)
     await room_store.write_if_missing(TEST_ROOM_ID, [ANOTHER_VALID_ACTION])
     assert list(await room_store.read(TEST_ROOM_ID)) == [VALID_ACTION]
+
```

Comment:
I think we need tests for zero rooms and tests for activity time for multiple rooms